packages/plugin-api-docs/src/plugin.ts

```
@@ -0,0 +1,351 @@
+} from "./excludes";
+import type { ContentSurfaceLike, FieldLike } from "./fields";
+import { generateOpenApiDocument, type ContentConfig } from "./generate";
+import { applyMountOverrides, type MountOverride } from "./mount-overrides";
+import { pluginRoutesToDocs } from "./plugin-routes";
+import { scanAppDirectory } from "./scan";
+// The Scalar standalone bundle, vendored as a build-time text asset. Imported
+// (not fs-resolved) because the host app's bundler pulls this dist into its
+// own graph: it statically analyzes literal require.resolve("@scalar/...")
+// and fails, and import.meta.url points into the bundle, not this package.
+// A string constant is inert data to every bundler.
+import scalarBundleSource from "./vendor/scalar-standalone.js.txt";
```

ESLint

[error] Unable to resolve path to module './vendor/scalar-standalone.js.txt'.

(import-x/no-unresolved)